Notes/BeetleBattleDesign.md

```diff
@@ -0,0 +1,115 @@
+Beetle Battle Design Specification
+==================================
+
+Beetlebergs, being a game about beetles battling, needs to have a design for how those beetles battle. This is that
+document.
+
+The Battle
+----------
+
+### The Arena
```

Comment:
Sounds good. I may have been overcomplicating things by getting octagon vertices off a GIS'd octagon image lol. Yeah let's keep things simple for now and just have a flat quadrilateral battlefield.

One important thing about the game re: the battlefield that didn't get shown in that video, if a round exceeds a certain time (dunno, maybe 30-60 seconds?) a circle begins to tighten, forcing the beetles closer together until there's one survivor.

Reply:
Oh, that might be a good feature that we add. I think that can be simulated as an "attack" that gets spawned at a certain time.